Sources/CIKernel+MetalSource.swift

```diff
@@ -0,0 +1,126 @@
+import CoreImage
+
+
+public extension CIKernel {
+
+    /// Errors that can be thrown by the Metal kernel runtime compilation APIs.
+    enum MetalKernelError: Swift.Error {
+        case functionNotFound(_ message: String)
+        case noMatchingKernelFound(_ message: String)
+        case blendKernelsNotSupported(_ message: String)
+        case ciklKernelCreationFailed(_ message: String)
+    }
+
+
+    /// Compiles a Core Image kernel at runtime from the given Metal `source` string.
+    ///
+    /// ⚠️ Important: There are a few limitations to this API:
```

Comment:
This could be a section with `- Important: `

Reply:
I tried, but I couldn't get it to work that the section then contains the bullet list. It was just not displayed at all for some reason... 🤷‍♂️